tool/grind.dart

```diff
@@ -2,37 +2,55 @@
 // governed by a BSD-style license that can be found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:io';
 
 import 'package:grinder/grinder.dart';
 
 main(args) => grind(args);
 
 @Task()
-void init(GrinderContext context) {
-  Pub.get(context);
-}
+void init(GrinderContext context) => defaultInit(context);
 
-@Task() @Depends(init)
+@Task()
+@Depends(init)
 void analyze(GrinderContext context) {
-  Analyzer.analyzePaths(context,
-      ['example/ex1.dart', 'example/ex2.dart']);
+  Analyzer.analyzePaths(context, ['example/grind.dart']);
   Analyzer.analyzePaths(context,
       ['lib/grinder.dart', 'lib/grinder_files.dart', 'lib/grinder_tools.dart']);
 }
 
-@Task() @Depends(init)
+@Task()
+@Depends(init)
 void tests(GrinderContext context) {
   Tests.runCliTests(context);
 }
 
-@Task() @Depends(init)
+@Task()
+@Depends(init)
 Future testsWeb(GrinderContext context) {
   return Tests.runWebTests(context, directory: 'web', htmlFile: 'web.html');
 }
 
-@Task() @Depends(init)
+@Task()
+@Depends(init)
 Future testsBuildWeb(GrinderContext context) {
   return Pub.buildAsync(context, directories: ['web']).then((_) {
     return Tests.runWebTests(context, directory: 'build/web', htmlFile: 'web.html');
   });
 }
+
+@Task('Analyze the generated grind script')
+@Depends(init)
+analyzeInit(GrinderContext context) {
+  Directory dir = new Directory('init_temp');
+
+  try {
+    dir.createSync();
+    File pubspec = new File('init_temp/pubspec.yaml');
```

Comment:
Might be good to use the `path` package.  I'm usually stuck on a windows machine, and I *think* it will work anyways with these paths, but not 100% sure.